utils/trivialize-rules/trivialize-targets.js

```diff
@@ -0,0 +1,252 @@
+"use strict";
+
+let util = require("util");
+let path = require("path");
+let xml2js = require("xml2js");
+
+let fs = require("graceful-fs");
+let readdir = util.promisify(fs.readdir);
+let readFile = util.promisify(fs.readFile);
+let parseString = util.promisify(xml2js.parseString);
+
+let chalk = require("chalk");
+let validUrl = require("valid-url");
+let escapeStringRegexp = require("escape-string-regexp");
+let { explodeRegExp, UnsupportedRegExp } = require("./explode-regexp");
+
+let rulesDir = "src/chrome/content/rules";
+
+(async () => {
+  let filenames = (await readdir(rulesDir)).filter(fn => fn.endsWith(".xml"));
+  let filePromises = filenames.map(async filename => {
+    function createTag(tagName, colour, print) {
+      return (strings, ...values) => {
+        let result = `[${tagName}] ${chalk.bold(filename)}: ${strings[0]}`;
+        for (let i = 1; i < strings.length; i++) {
+          let value = values[i - 1];
```

Comment:
```
        for (let i = 0; i < strings.length - 1; i++) {
          let value = values[i];
```

Reply:
in fact, this part of code is copied from `trivialize-rules.js`. I believe the index is correct because `strings[0]` is already used above while `strings[i]` and `values[i - 1]` works like key-value pairs.